### ISSUE TITLE
Column Picker - Notify on column hide/show

### ISSUE THREAD

**Issue by Quantas** [Quantas/shark-ng-table#57]
When checking/unchecking the choose columns choices, there's no notification that anything happened. Ideally, a screenreader user should be notified that a certain column was hidden or shown.
